### ISSUE TITLE
[Feature] Display SoundCloud user likes as playlist

### ISSUE THREAD

**Issue by neopostmodern** [omega-gg/MotionBox#44]
Right now it's not possible to view the likes of a SoundCloud user as a playlist, as seen online at https://souncloud.com/USERNAME/likes – but it would be a useful way of browsing one's own or a friend's music outside of the officiall app/website.

**Comment by 3unjee** [comment]
Yes, the https://github.com/omega-gg/backend/blob/master/soundcloud.vbml backend could probably be patched to handle this. I'm open to contributions on that matter.
